drivers/sensor/bosch/bmp180/Kconfig

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2024 Chris Ruehl
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig BMP180
+	bool "Bosch BMP180 pressure sensor"
+	default y
+	depends on DT_HAS_BOSCH_BMP180_ENABLED
+	select I2C if $(dt_compat_on_bus,$(DT_COMPAT_BOSCH_BMP180),i2c)
+	help
+	  Enable driver for the Bosch BMP180 pressure sensor
+
+if BMP180
+
+config BMP180_OSR_RUNTIME
+	bool "Change OSR at runtime."
+	default y
```

Comment:
Matter of preference but I would not use a if but rather a depends on for one single option

Reply:
Btw. this is missing a help text.

Reply:
add the help text. thanks